contrib/buildrpm.sh

```diff
@@ -39,6 +42,9 @@ EOF
     shift
 done
 
+if [ $opt_no_dist -eq 1 ]; then
+    rpmmacros="$rpmmacros '--undefine=dist'"
```

Comment:
@amaslenn - is it possible to undefine or zero dist inside spec file for %?fedora only? and keep all distro-specific limitations inside spec file?